ReviewChecklist.md

```diff
@@ -34,6 +40,24 @@ It is also meant to serve as a final checklist for reviewers to go through befor
     - Review updated files before committing them.
         **Are expectations correct and do updated tests still serve their purpose?**
 
+## Abandoned PRs
+- [ ] **Is the submitter still responsive?**
+    - If the PR had no activity from the submitter in the last 3 weeks (despite receiving reviews and our prompts) we consider it abandoned.
```

Comment:
This is something we should narrow to 2 weeks in the near future as we're ramping up our reviewing capacity.

Reply:
Changed back to 2 weeks. Some current PRs are well beyond even the 2 week limit so we have to treat them specially anyway - no reason not to have the final rule here already.